frameworks/Rust/viz/src/db_pg.rs

```diff
@@ -166,20 +168,18 @@ impl PgConnection {
             params.push(&w.id);
         }
 
-        let st = self.updates[(num as usize) - 1].clone();
+        let st = self.updates[num - 1].clone();
 
         self.client.query(&st, &params[..]).await?;
 
         Ok(worlds)
     }
 
-    pub async fn tell_fortune(&self) -> Result<Vec<Fortune>, PgError> {
-        let mut items = Vec::with_capacity(32);
-
-        items.push(Fortune {
+    pub async fn tell_fortune(&self) -> Result<SmallVec<[Fortune; 32]>, PgError> {
+        let mut items: SmallVec<[_; 32]> = smallvec::smallvec![Fortune {
```

Comment:
I did not see this the first time through. Fortunes must be held in dynamically sized data structures. [See Rule .v](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#fortunes)